src/components/header/PatientHeader.jsx

```diff
@@ -0,0 +1,215 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+// import { fetchPatientRecord, fetchPatientNote } from '../../actions/patient';
+
+import dateFns, { distanceInWordsToNow } from 'date-fns';
+
+
+export class PatientHeader extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showContactInfo: false,
+    };
+  }
+
+  componentDidMount() {
+    // TODO: would need to run a saga to get initiate this, for now just pass props
```

Comment:
It's a good point... also, since this is a header, it should really only be used on a page that is loaded in the context of patient, so it may not need to actually fetch the patient at all... the component above it might always need to responsible for providing the full patient object?

Reply:
It just occurred to me... we've created a Patient domain object in react-components to provide a standardized, simplified version of a Patient... we should update this header to work with that representation... doesn't need to be done immediately, though...

https://github.com/openmrs/openmrs-react-components/blob/master/src/domain/Patient.js

Reply:
Great, let me know if you have any questions or suggestions about it!